code/modules/mob/living/simple_animal/abnormality/he/blue_shepherd.dm

```diff
@@ -287,13 +291,23 @@
 		return FALSE
 	return ..()
 
+/mob/living/simple_animal/hostile/abnormality/blue_shepherd/proc/targetslash()
```

Comment:
isn't the only different line the `addtimer()`?
If so, i dont really see a point making an entirelly different proc, and it should be a var in the proc instead like
```
/mob/living/simple_animal/hostile/abnormality/blue_shepherd/proc/slash(fast = FALSE)
```